connector/openshift/openshift.go

```diff
@@ -21,31 +22,39 @@ import (
 	"github.com/dexidp/dex/storage/kubernetes/k8sapi"
 )
 
+const (
+	wellKnownURLPath = "/.well-known/oauth-authorization-server"
+	usersURLPath     = "/apis/user.openshift.io/v1/users/~"
+)
+
 // Config holds configuration options for OpenShift login
 type Config struct {
-	Issuer       string   `json:"issuer"`
-	ClientID     string   `json:"clientID"`
-	ClientSecret string   `json:"clientSecret"`
-	RedirectURI  string   `json:"redirectURI"`
-	Groups       []string `json:"groups"`
-	InsecureCA   bool     `json:"insecureCA"`
-	RootCA       string   `json:"rootCA"`
+	Issuer          string   `json:"issuer"`
+	ClientID        string   `json:"clientID"`
+	ClientSecret    string   `json:"clientSecret"`
+	RedirectURI     string   `json:"redirectURI"`
+	Groups          []string `json:"groups"`
+	InsecureCA      bool     `json:"insecureCA"`
+	RootCA          string   `json:"rootCA"`
+	TrustedCertPool *x509.CertPool
```

Comment:
I've now more sided with the boolean flag to specify whether or not to include system roots within #3 .
It will include the `x509.SystemCertPool` by default, as beforehand. When specifying `rootCA` **and not** specifying `useSystemRootCAs` the behavior will stay the same.
Only when specifically choosing the `useSystemRootCAs` in combination with `rootCA` they will be added.

I'm not sure this is the _best_ way to achieve it, and probably I will rename it from `useSystemRootCAs` to something more clear, but at the very least this doesn't change the current default behavior